apps/masterbots.ai/components/routes/thread/user-thread-panel.tsx

```
@@ -160,7 +160,17 @@ export default function UserThreadPanel({
     }
   }, [continuousThreadId, session])
 
-  const threads = state.threads.length > initialThreads.length ? state.threads : initialThreads
+  useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: isAdminMode

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)